.circleci/config.yml

```diff
@@ -3,7 +3,10 @@ version: 2.1
 jobs:
   test_tool_and_runtime_java:
     docker:
-      - image: cimg/openjdk:8.0
+      - image: cimg/openjdk:11.0.13
```

Comment:
I recommend waiting for the merging of that request https://github.com/antlr/antlr4/pull/3407 There are some troubles with JDK updating.

Reply:
Almost ready.

Reply:
looks like we are going to be only java 8 for this next release because there's no way to use java 11 and generate java 8 class files apparently, at least reliably.